packages/actor-http-native/lib/Requester.ts

```diff
@@ -56,14 +68,24 @@ export default class Requester {
         response.pipe(decoded);
         // Copy response properties
         decoded.statusCode = response.statusCode;
-        decoded.headers = response.headers;
+        // Wrap headers into an header object type
+        const responseHeaders: any = new Headers();
```

Comment:
Duplicated code should be avoided at all cost (see code starting at line 84).
I would suggest abstracting this into a method such as `convertFetchHeadersToHash`.